tests/foreman/api/test_contentview.py

```diff
@@ -19,6 +21,36 @@
 REPEAT = 3
 
 
+def _check_bz_1186432(humanized_errors):
+    """Check whether any error messages appear to be due to BZ 1186432.
+
+    This is an example of ``humanized_errors`` from a server suffering from BZ
+    1186432:
+
+    > [u'ERF12-4115 [ProxyAPI::ProxyException]: Unable to get classes from
+    > Puppet for example_env ([RestClient::NotAcceptable]: 406 Not
+    > Acceptable) for proxy https://<snip>:9090/puppet']
+
+    :param list humanized_errors: A list of strings. This list can be extracted
+        from a response like so: ``response['humanized']['errors']``.
+    :returns: Nothing.
+    :rtype: None
+    :raises: ``unittest.SkipTest`` if any of the strings in
+        ``humanized_errors`` look suspicious.
+
+    """
+    bz_1186432_re = (
+        r'ERF12-4115 \[ProxyAPI::ProxyException\].*'
+        r'\[RestClient::NotAcceptable\]: 406 Not Acceptable'
+    )
+    for error in humanized_errors:
+        if (
+                re.search(bz_1186432_re, error) is not None
+                and bz_bug_is_open(1186432)
+        ):
```

Comment:
This can be made more compact still maintaining readability

```py
        if (re.search(bz_1186432_re, error) is not None and
                bz_bug_is_open(1186432)):
            raise SkipTest('BZ 1186432 is open: {0}'.format(error))
```